tests/attestation/sgxlkl_attests_to_oe/oe_enclave/enc/makefile

```diff
@@ -0,0 +1,42 @@
+# Detect C and C++ compiler options
+# if not gcc and g++, default to clang-7
+C_COMPILER=$(notdir $(CC))
+ifeq ($(C_COMPILER), gcc)
+        CXX_COMPILER=$(notdir $(CXX))
+        USE_GCC = true
+endif
+
+ifeq ($(USE_GCC),)
+        CC = clang-7
+        CXX = clang++-7
+        C_COMPILER=clang
+        CXX_COMPILER=clang++
+endif
+
+CFLAGS=$(shell pkg-config oeenclave-$(C_COMPILER) --cflags)
+CXXFLAGS=$(shell pkg-config oeenclave-$(CXX_COMPILER) --cflags)
+LDFLAGS=$(shell pkg-config oeenclave-$(CXX_COMPILER) --libs)
+INCDIR=$(shell pkg-config oeenclave-$(C_COMPILER) --variable=includedir)
+ATTESTATION_COMMONS="../../../common"
+
+.PHONY: all build clean run
+
+all:
+	$(MAKE) build
+	$(MAKE) sign
+
+build:
+	@ echo "Compilers used: $(CC), $(CXX)"
+	oeedger8r ../tlssrv.edl --trusted \
+		--search-path $(INCDIR) \
+		--search-path $(INCDIR)/openenclave/edl/sgx
+	$(CXX) -c $(CXXFLAGS) $(INCLUDES) -std=c++11 tlssrv.cpp
+	$(CXX) -c $(CFLAGS) $(CINCLUDES) $(ATTESTATION_COMMONS)/gencreds.cpp
+	$(CC) -c $(CFLAGS) $(CINCLUDES) tlssrv_t.c
+	$(CXX) -o tlssrv_enc tlssrv.o gencreds.o tlssrv_t.o $(LDFLAGS) -lmbedtls -lmbedcrypto -loehostsock -loehostresolver -loecore -loelibc -loesyscall
+
+sign:
+	oesign sign -e tlssrv_enc -c enc.conf -k private.pem
```

Comment:
Since the sgx-lkl enclave doesn't care about the MRSIGNER of oe enclave, we can get rid of the below private.pem file and generate a random private key here.

Reply:
use the following before oesign:
```
openssl genrsa -out private.pem -3 3072
```

Reply:
@jxyang I removed private key and use openssl to generate one. Let me know if that looks good to you